hyungjin/app.js

```diff
@@ -0,0 +1,64 @@
+
+const http =require ("http");
+const express = require("express");
+const cors = require("cors");
+const morgan =  require("morgan");
+const dotenv = require("dotenv");
+
+const { DataSource } = require('typeorm') 
+// .env 파일에 저장해놓은 환경 변수들을 불러와 DataSource 명령어를 통해서 db와의 커넥션 환경 세팅을 완료
+
+dotenv.config()
+//환경변수에 적어놓앗던 typeorm을 app.js에서 활용할수있게 발동시키는작업
```

Comment:
- origin에 작업 내용을 올릴 땐 가급적 개발 시에 작성한 주석은 지워주세요!
- import 순서 컨벤션에 맞게 수정해주세요! 사소한 부분이라 놓치기 쉽지만 처음 코드를 작성할 때부터 신경쓰고 작성하게 되면 추후에 관리하기도 용이하며 가독성도 좋아집니다.
- 다음과 같은 순서로 import 해줍니다.
    - Bulit-in 패키지
    - 3rd-party 패키지
    - custom 패키지
    - 
    - 
```suggestion
//// Built-in package
const http = require('http')

//// 3rd-party package
require('dotenv').config();
const express = require('express');
const cors    = require('cors');
const morgan  = require('morgan');
const { DataSource } = require('typeorm');

//custom package
const api = require('./../api')

const app = express();
```